### PR TITLE
feat: replace critical field on tasks with priority

### PR DESCRIPTION
**Pull request by m4thayus**
Screenshot 2024-09-19 at 1 26 05 PM

### CODE REVIEW

File: app/assets/stylesheets/histogram.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      }
+      &.bar {
+        width: 1em;
+        &.fill {
+          background-color: black;
+
+        }
+      }
+    }
+
```
Comment (scottb) [inline app/assets/stylesheets/histogram.scss:25]: Blank line?

File: app/assets/stylesheets/main.scss [hunk 2]
```diff
@@ -66,6 +66,65 @@ table {
...
+        span {
+          border-right: solid black 1px;
+          height: 100%;
+          width: 100%;
+          &:last-child {
+            border-right: unset;
+          }
+        }
+      }
+
```
Comment (scottb) [inline app/assets/stylesheets/main.scss:126]: Blank line?

File: app/controllers/tasks_controller.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@ class TasksController < ApplicationController
   load_and_authorize_resource except: %i[index]
   before_action :associate_board
 
-  def index
+  def index # rubocop:disable Metrics/AbcSize
```
Comment (scottb) [inline app/controllers/tasks_controller.rb]: This should be either fixed, or turned off for the whole app.

File: app/views/tasks/index.html.slim [hunk 4]
```diff
@@ -9,6 +9,9 @@
       li
         a[href="#available"] Available
 
+- if @histogram.present?
+  = render "histogram", counts: @histogram, max: @histogram.max { |a, b| a.last <=> b.last }.last
```
Comment (scottb) [inline app/views/tasks/index.html.slim:13]: `max: @histogram.max_by(&:ast).last`

**Review by scottb — APPROVED** [review]

**Review by JlordA — APPROVED** [review]

**Pull request merged**